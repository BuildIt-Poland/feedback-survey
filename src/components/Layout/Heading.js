import styled from 'styled-components';
import { Heading } from 'grommet';

const StyledHeading = styled(Heading)`
  font-weight: 400;
  text-align: center;
`;

export default StyledHeading;
