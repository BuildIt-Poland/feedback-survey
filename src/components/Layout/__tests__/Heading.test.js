import React from 'react';
import { create } from 'react-test-renderer';

import Heading from '../Heading';

describe('COMPONENT - Layout Heading', () => {
  it('renders correctly', () => {
    const component = create(<Heading />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
