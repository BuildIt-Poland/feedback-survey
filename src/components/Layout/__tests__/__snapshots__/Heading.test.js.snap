// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Layout Heading renders correctly 1`] = `
<h1
  className="sc-bdVaJa hFrVFs StyledHeading-sc-1rdh4aw-0 jVpYfH"
/>
`;
